app/views/bets/show.html.erb

```diff
@@ -1,49 +1,16 @@
-<div class="container">
-  <hr>
-  <div class="row">
-    <div class="col col-lg-12">
-      <h2>Viewing bet: <%= @bet.id %></h2>
-    </div>
-  </div>
-  <hr>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>ID</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.id %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Status</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.status %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Creator ID</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.creator_id %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Expires at</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.expires_at %></h4>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col col-lg-4">
-      <h4>Text</h4>
-    </div>
-    <div class="col col-lg-8">
-      <h4><%= @bet.bet %></h4>
+<hr>
+<div class="row">
+  <div class="col col-lg-6 offset-lg-3">
+    <div class="card text-center">
+      <div class="card-header text-muted">
+        <h4><%= @bet.creator.username %></h4>
```

Comment:
Is creator a required association on bet? What would happen here if a bets creator was `nil`? Same thing with username, I am not seeing any validations on a users username. What if they didn't have a user name?